docs/getting-started/install.md

```diff
@@ -310,6 +310,7 @@ and their default values.
 | `personas.deploy`                | Install roles and bindings for Crossplane user personas         | `true`
 | `templateStacks.enabled`         | Enable experimental template stacks support                     | `true`
 | `templateStacks.controllerImage` | Template Stack controller image                                 | `crossplane/templating-controller:v0.2.1`
+| `priorityClassName      `        | Priority class name for crossplane and stack manager pods       | `""`
```

Comment:
Nitpick: Would you mind updating `forceImagePullPolicy` default value to also be `""` below for consistency? It looks much nicer than `` :)

Reply:
Sure, done.